src/Components/Panels/Toolbox/PassabilityControl.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ChangeEvent } from 'react';
+
+import { IPassableObject } from '../../../Document/Attributes/IPassableObject';
+
+import styles from './PassabilityControls.module.scss';
+
+interface Props {
+  data: IPassableObject;
+  onChange: (changes: IPassableObject) => void;
+}
+
+const PassabilityControls = ({ data, onChange }: Props) => {
+  const handleDriveThroughOnChange = (_: ChangeEvent) => {
+    const drivethrough = !data.drivethrough;
+    onChange({
+      ...data,
+      drivethrough,
+    });
+  };
+  const handleShootThroughOnChange = (_: ChangeEvent) => {
+    const shootthrough = !data.shootthrough;
+    onChange({
+      ...data,
+      shootthrough,
+    });
+  };
+
+  return (
+    <div className={styles.wrapper}>
```

Comment:
```suggestion
    <fieldset className={styles.wrapper}>
```

Let's make this a [`<fieldset>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset). It's useful for grouping related input fields.

Semi-related: the [`<legend>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend) tag can **only** exist inside of a `<fieldset>`; since you had it inside of a `<div>`, it doesn't make semantically.